packages/webpack/react-webpack-plugin/test/remove-background-only-loader.js

```diff
@@ -0,0 +1,7 @@
+// Copyright 2024 The Lynx Authors. All rights reserved.
+// Licensed under the Apache License Version 2.0 that can be found in the
+// LICENSE file in the root directory of this source tree.
+
+export default function removeBackgroundOnly(source) {
+  return source.replace(/import\s+['"]background-only['"];\n?/g, '');
```

Comment:
Instead of removing `background-only`, can we simple add `background-only` as the dependencies of `@lynx-js/react`?

Reply:
Yeah, I mean "as the dependencies of `@lynx-js/react`"

Reply:
Sorry, I got it, it works great! I have updated to use a `background-only` package as `devDep` of `@lynx-js/react`.